### PR TITLE
Scala grammar fix for #1625 & #1660

### PR DESCRIPTION
**Pull request by gokulvenkats**
Hi,

@KvanTTT and @studentmain I have raised a PR for the Issue No. 1625 & 1660. Kindly verify.

### CODE REVIEW

File: scala/Scala.g4 [hunk 1]
```diff
@@ -186,10 +186,15 @@ exprs
...
-   | NL? blockExpr
-   ;
+    : '(' args ')'
+    | '{' args '}'
+    | NL? blockExpr
+    ;
+
+args
+    : exprs?
+    | (exprs ',')? postfixExpr ':' '_' '*' // should be (':' | '_' | '*') ?
```
Comment (studentmain) [inline scala/Scala.g4:196]: If you know the correct syntax, please update it and remove this comment. Or add a TODO here, so someone can fix it later.

Comment (gokulvenkats) [reply scala/Scala.g4]: Hi @studentmain ,

I have did the correction. Can you verify & close the PR.

**Comment by teverett** [comment]
@gokulvenkats  thanks

**Comment by gokulvenkats** [comment]
Thanks! @teverett

**Pull request merged**